admin/query/src/test/java/org/codice/ditto/replication/admin/query/ReplicationUtilsTest.java

```diff
@@ -415,27 +522,47 @@ public void getReplications() throws Exception {
 
     ReplicationStatus status = new ReplicationStatusImpl("test");
     status.setPushCount(1L);
-    status.setPushBytes(1024 * 1024 * 5L);
+    status.setPushBytes(5L);
     status.setPullCount(2L);
-    status.setPullBytes(1024 * 1024 * 10L);
+    status.setPullBytes(10L);
+    status.setLastRun(LAST_RUN);
+    status.setLastSuccess(LAST_SUCCESS);
+    status.setStartTime(START_TIME);
+    status.setDuration(9L);
+    status.setPullFailCount(99L);
+    status.setPushFailCount(100L);
+    status.setStatus(Status.SUCCESS);
     when(history.getReplicationEvents("test")).thenReturn(Collections.singletonList(status));
     when(replicator.getActiveSyncRequests()).thenReturn(Collections.emptySet());
     ReplicatorConfigImpl config = new ReplicatorConfigImpl();
-    config.setId("id");
+    config.setId(REP_ID);
     config.setName("test");
     config.setFilter("cql");
     config.setSource("srcId");
     config.setDestination("destId");
     config.setBidirectional(false);
     when(configManager.objects()).thenReturn(Stream.of(config));
+
+    // when
     ReplicationField field = utils.getReplications(false).getList().get(0);
+
+    // then
     assertThat(field.name(), is("test"));
     assertThat(field.source().id(), is("srcId"));
     assertThat(field.destination().id(), is("destId"));
     assertThat(field.filter(), is("cql"));
     assertThat(field.biDirectional(), is(false));
```

Comment:
I don't think it needs to be done in this PR but at some point we should probably clean these tests up.